### PR TITLE
Add openscap by ansible docs

### PR DESCRIPTION
**Pull request by xprazak2**
Draft of docs for https://github.com/theforeman/foreman_openscap/pull/367

### CODE REVIEW

File: plugins/foreman_openscap/0.12/index.md [hunk 1]
```diff
@@ -0,0 +1,628 @@
...
+* /etc/pki/consumer/key.pem
+
+
+### 4.6.2 Manual policy deployments
+
+If a user chooses `Manual` as a policy deployment, Foreman will not care about client configuration and will leave it entirely up to user to configure clients. Multiple manual steps will be required to configure clients to send reports (which can be automated by using Ansible or Puppet).
+
+1) Install foreman_scap_client package on the client.
+
+2) Create client config in /etc/foreman_scap_client/config.yaml and populate it with appropriate values. See example config in section [6.3](plugins/foreman_openscap/0.9/index.html#6.3Configforclient). The most convenient way to get the attributes for policies is to add a host parameter with `<%= @host.policies_enc %>` value to your client and then get the JSON value it generates from ENC.
```
Comment (ares) [inline plugins/foreman_openscap/0.12/index.md:391]: we should provide API endpoint and hammer command to generate this more easily and/or possibly display this on host detail page, should I open RFE?

Comment (xprazak2) [reply plugins/foreman_openscap/0.12/index.md:391]: Yes, it would be nice to have.

Comment (ares) [reply plugins/foreman_openscap/0.12/index.md:391]: opened https://projects.theforeman.org/issues/26324

**Comment by ares** [comment]
@xprazak2 anything else to add? 0.12.0 release is available

**Comment by xprazak2** [comment]
No, it should be all.

**Comment by ares** [comment]
Thanks @xprazak2, merging!

**Pull request merged**